Unix/provmgr/provmgr.c

```diff
@@ -416,11 +416,13 @@ static MI_Result MI_CALL _GetProviderByClassName(
     _In_ ProvMgr* self,
     _In_ const ProvRegEntry* proventry,
     _In_ MI_ConstString cn,
+    _Inout_ MI_Char** errmsg,
```

Comment:
This is an _Out_. Please move to after the provOut parameter. Will need to update all usages again, sorry.